lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -859,180 +855,6 @@ def _create_from_store(
-        invocation_step = self.workflow_manager.get_invocation_step(trans, decoded_invocation_step_id)
-        return self.__encode_invocation_step(trans, invocation_step)
-
-    @expose_api_anonymous_and_sessionless
-    def invocation_step_jobs_summary(self, trans: GalaxyWebTransaction, invocation_id, **kwd):
-        """
-        GET /api/workflows/{workflow_id}/invocations/{invocation_id}/step_jobs_summary
-        GET /api/invocations/{invocation_id}/step_jobs_summary
-
-        return job state summary info aggregated across per step of the workflow invocation
-
-        Warning: We allow anyone to fetch job state information about any object they
-        can guess an encoded ID for - it isn't considered protected data. This keeps
-        polling IDs as part of state calculation for large histories and collections as
-        efficient as possible.
-
-        :param  invocation_id:    the invocation id (required)
-        :type   invocation_id:    str
-
-        :rtype:     dict[]
-        :returns:   an array of job summary object dictionaries for each step
-        """
-        decoded_invocation_id = self.decode_id(invocation_id)
-        ids = []
-        types = []
-        for job_source_type, job_source_id, _ in invocation_job_source_iter(trans.sa_session, decoded_invocation_id):
-            ids.append(job_source_id)
-            types.append(job_source_type)
-        return [self.encode_all_ids(trans, s) for s in fetch_job_states(trans.sa_session, ids, types)]
-
-    @expose_api_anonymous_and_sessionless
-    def invocation_jobs_summary(self, trans: GalaxyWebTransaction, invocation_id, **kwd):
-        """
-        GET /api/workflows/{workflow_id}/invocations/{invocation_id}/jobs_summary
-        GET /api/invocations/{invocation_id}/jobs_summary
-
-        return job state summary info aggregated across all current jobs of workflow invocation
-
-        Warning: We allow anyone to fetch job state information about any object they
-        can guess an encoded ID for - it isn't considered protected data. This keeps
-        polling IDs as part of state calculation for large histories and collections as
-        efficient as possible.
-
-        :param  invocation_id:    the invocation id (required)
-        :type   invocation_id:    str
-
-        :rtype:     dict
-        :returns:   a job summary object merged for all steps in workflow invocation
-        """
-        ids = [self.decode_id(invocation_id)]
-        types = ["WorkflowInvocation"]
-        return [self.encode_all_ids(trans, s) for s in fetch_job_states(trans.sa_session, ids, types)][0]
-
-    @expose_api
-    def update_invocation_step(self, trans: GalaxyWebTransaction, invocation_id, step_id, payload, **kwd):
-        """
-        PUT /api/workflows/{workflow_id}/invocations/{invocation_id}/steps/{step_id}
-        PUT /api/invocations/{invocation_id}/steps/{step_id}
-
-        Update state of running workflow step invocation - still very nebulous
```

Comment:
@jmchilton is this less nebulous now?  IUUC currently this takes the `action` bool param and either unpauses or not -- is there more to it? I'd like to improve the documentation here.